### PR TITLE
Add flake8 & isort

### PR DESCRIPTION
**Pull request by maxiberta**

### CODE REVIEW

File: acceptable/util.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def _sort_schema(schema):
             else:
                 yield v
     else:
-        yield d
+        yield schema
```
Comment (maxiberta) [inline acceptable/util.py:53]: Variable `d` doesn't exist. The only variable in this scope is `schema` and it does make sense to return it as is.

File: acceptable/tests/test_util.py [hunk 2]
```diff
@@ -10,10 +10,10 @@ def test_sort_schema_simple(self):
         srtd = util.sort_schema({"5": "5", "1": "1", "3": "3"})
         self.assertEqual(["1", "3", "5"], list(srtd))
 
-    def test_sort_schema_simple(self):
+    def test_sort_schema(self):
         d = {"5": "5", "1": "1", "3": "3"}
-        l = [5, 1, 3]
-        srtd = util.sort_schema({"foo": {"b": [d], "a": [l]}})
+        ll = [5, 1, 3]
```
Comment (maxiberta) [inline acceptable/tests/test_util.py:15]: Flake8 doesn't like single letter variable names.

**Review by suligap — APPROVED** [review]
+1 thanks!

**Review by shanepelletier — APPROVED** [review]
LGTM!

**Pull request merged**